src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java

```diff
@@ -3,16 +3,17 @@
 import java.util.Calendar;
 import java.util.Date;
 import java.util.GregorianCalendar;
+import java.util.TimeZone;
 
 public class ExpiryCalendar {
     /**
      * Hard-coded to 7 days.
      * @return
      */
     public static Date getDeadlineExpiryDate() {
-        Calendar calendar = new GregorianCalendar();
+        Calendar calendar = new GregorianCalendar(TimeZone.getTimeZone("UTC"));
         calendar.add(Calendar.DAY_OF_YEAR, 7);
-        calendar.set(Calendar.HOUR, 23);
+        calendar.set(Calendar.HOUR_OF_DAY, 23);
         calendar.set(Calendar.MINUTE, 59);
         calendar.set(Calendar.SECOND, 59);
```

Comment:
It's probably worth setting the millis part to zero, just so that we're using a consistent date time.